### DISCUSSION TITLE
Bug: Lagging

### DISCUSSION THREAD

**Discussion by Annguyen1qqq** [eritislami/evobot#1049] [General]
The bot is lagging when it plays music. 
Does anyone have any solution?

**Comment by SugarD-x** [comment]
On my end, initially playing the music seems to lag because it takes a very long time to load the icons for the music controls, which popped up several versions back. As far as lag during actual ongoing play, I only notice it from time to time, which appears to be some kind of connection issue causing the stream to hiccup for a slight moment.

**Comment by legouju** [comment]
What are you hosting the bot on, free or paid/your own, as generally free hosting services have limited/slower resources.

**Comment by SugarD-x** [comment]
When it happens to me, I'm on a paid VPS with a very good connection for both up and down.
